<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetYT</string>
  </property>
  <widget class="QLabel" name="urlLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>URL: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlContainer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="videoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Video List</string>
   </property>
   <widget class="QListWidget" name="videoList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>481</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="settingsBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLabel" name="destLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destContainer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <zorder>settingsBox</zorder>
  <zorder>urlLabel</zorder>
  <zorder>urlContainer</zorder>
  <zorder>addButton</zorder>
  <zorder>videoBox</zorder>
  <zorder>downloadButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
